File: programming/lab2/src/main/java/pokemons/Leavanny.java
package pokemons;

import moves.status.SwordsDance;

public class Leavanny extends Swadloon {
    public Leavanny(String name, int level) {
        super(name, level);
        this.setStats(75, 103, 80, 70, 80, 92);
        this.addMove(new SwordsDance());
    }

    public Leavanny() {
        super("Leavanny", 1);
    }
}

File: programming/lab2/app/src/main/kotlin/org/itmo/pokemons/Leavanny.kt
package org.itmo.pokemons

import org.itmo.moves.status.SwordsDance

class Leavanny(name: String, level: Int) : Swadloon(name, level) {
    init {
        addMove(SwordsDance())
        setStats(75.0, 103.0, 80.0, 70.0, 80.0, 92.0)
    }
}